pysrc/juliacall/__init__.py

```diff
@@ -50,11 +50,29 @@ def backtrace(self):
 
 def init():
     import os
+    if "CONDA_PREFIX" in os.environ: #It does only when calling from PythonCall
+        ## For Windows and Python 3.8+ we need to add dll search paths
+        import sys
+        if os.name == "nt" and sys.hexversion >= 0x308000:
+            conda_environment_path = os.environ["CONDA_PREFIX"]
+            dll_search_paths = [
+                conda_environment_path,
+                os.path.join(conda_environment_path, "Library", "mingw-w64", "bin"),
+                os.path.join(conda_environment_path, "Library", "usr", "bin"),
+                os.path.join(conda_environment_path, "Library", "bin"),
+                os.path.join(conda_environment_path, "Scripts"),
+                os.path.join(conda_environment_path, "bin")
```

Comment:
Is there somewhere these are documented as potentially containing DLLs? Does the `CONDA_PREFIX` folder itself ever contain them?

Reply:
I thought that all these folders would contain DLL files based on the `ENV["PATH"]`. I checked them individually and most of them do.

Reply:
For now keeping only `CONDA_PREFIX\\Library\\bin`